src/util/fhir.js

```diff
@@ -80,10 +80,21 @@ function createMedicationDispenseFromMedicationRequest(medicationRequest) {
   return medicationDispense;
 }
 
+function createMedicationFromMedicationRequest(medicationRequest) {
+  let medication = {};
+  medication.resourceType = 'Medication';
+  medication.id = medicationRequest?.id + '-med';
+  if (medicationRequest.medicationCodeableConcept) {
```

Comment:
We probably still should handle when the MedicationRequest contains a medicationReference instead of a medicationCodeableConcept. We should grab the Medication from the reference so we can get the code.

Reply:
It looks like we have a test case for this. The Turalio MedicationRequest (pat036-mr-turalio) for Alice Smith (pat036) contains a medicationReference instead of a medicationCodeableConcept.

Reply:
Made a bug ticket for this: https://jira.mitre.org/browse/REMS-668